docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -0,0 +1,353 @@
++++
+title = "FMW domains"
+date = 2019-04-18T06:46:23-05:00
+weight = 5
+pre = "<b> </b>"
++++
+
+Starting with release 2.2, the operator supports FMW Infrastructure domains.
+This means domains that are created with the FMW Infrastructure installer rather than the WebLogic
+installer.  These domains contain the Java Required Files (JRF) feature, and are 
+the pre-requisite for "upper stack" products like Oracle SOA Suite for example.
+These domains also require a database and the use of the Repository
+Creation Utility (RCU). 
+
+This section provides details about the special considerations for running
+FMW Infrastructure domains with the operator.  Other than those considerations
+listed here, FMW Infrastructure domains work in the same way as WebLogic domains.
+That is, the remainder of the documentation in this site applies equally to FMW 
+Infrastructure domains and WebLogic domains. 
+
+FMW Infrastructure domains are supported using both the "domain on a persistent volume" 
+and the "domain in a Docker image" [models]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}).
+If you plan to experiment with upper stack products (which are not officially supported
+by the operator yet) we strongly recommend using the domain on a persistent
+volume approach. 
+
+#### Limitations
+
+Compared to running a WebLogic Server domain in Kubernetes using the operator, the
+following limitations currently exist for FMW Infrastructure domains: 
+
+* The [WebLogic Logging Exporter](https://github.com/oracle/weblogic-logging-exporter)
+  currently only supports WebLogic server logs.  Other logs will not be sent to
+  Elasticsearch.  Note however that you can use a sidecar with a log handling tool
+  like Logstash or fluentd to get logs. 
+* The [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter)
+  currently only supports the WebLogic MBean trees.  Support for JRF MBeans has not
+  been added yet.
+* Only configured clusters are supported.  Dynamic clusters are not supported for
+  FMW Infrastructure domains.  Note that you can still use all of the scaling features,
+  you just need to define the maximum size of your cluster at domain creation time.
+* FMW Infrastructure domains are not supported with any version of the operator
+  before version 2.2.
+
+#### Creating a FMW Infrastructure Docker image
+
+You will need to create a Docker image containing the FMW Infrastructure binaries.
+A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure)
+is provided in the Oracle GitHub account that demonstrates how to create a Docker image
+to run FMW Infrastructure.  
+
+After cloning the repository and downloading the installer from Oracle Technology Network
+or e-delivery, you would create your image by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/dockerfiles
+./buildDockerImage.sh -v 12.2.1.3 -g 
```

Comment:
In the wiki http://aseng-wiki.us.oracle.com/asengwiki/display/ASDevWLS/Support+for+JRF-enabled+domains , you mentioned we need to change the Dockerfile to comment out the VOLUME_DIR as  the following:
`RUN mkdir -p /u01 && \
    chmod a+xr /u01 && \
    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle
#    mkdir -p $VOLUME_DIR
...
 # VOLUME  $VOLUME_DIR`

Do we still need to do that? In my test, I still commented  out the VOLUME_DIR in the new Docker file, seems working.

Reply:
seems to be working for me as-is - will test again